loader.js

```diff
@@ -190,12 +189,12 @@ var Module = null;
        }
 
        // and a count, then fetch them in
-       var len = urls.length;
+       len = urls.length;
        for (i = 0; i < len; i++) {
-         var node = urls[i],
-             drive = node.nodeName.split('_')[2],
-             title = 'Game File ('+ (i+1) +' of '+ (game ? len+1 : len) +')',
-             url = get_zip_url(node.textContent);
+         node = urls[i],
```

Comment:
Yeah, and most current browsers already implement let correctly, maybe we can use it...